src/Breadcrumb/BreadcrumbStep.tsx

```diff
@@ -0,0 +1,35 @@
+/* eslint react/prop-types: 0 */
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+
+import { BreadcrumbStep as _BreadcrumbStep, BreadcrumbLink as _BreadcrumbLink } from './styled';
+
+export type LocalBreadcrumbStepProps = {
+  color?: string;
+  href: string;
```

Comment:
We might want to make this optional if we don't require it to be linked all the time
```suggestion
  href?: string;
```

Reply:
for some reason I still can't figure out..after addressing PR suggestions, I've got this completely new error in console....any idea what can it be 😫 😫 😫 
 
![giphy](https://user-images.githubusercontent.com/41710405/52175620-8220b680-27fa-11e9-9bc9-8316b3f124fc.gif)

Reply:
👇 👇 👇  this one:
<img width="1066" alt="screen shot 2019-02-03 at 9 21 58 pm" src="https://user-images.githubusercontent.com/41710405/52175644-c14f0780-27fa-11e9-9359-d5fcbf5d61b6.png">